docs/topics/collection-aggregate.md

```diff
@@ -77,8 +77,8 @@ fun main() {
 //sampleStart
     val numbers = listOf(5, 2, 10, 4)
 
-    val sum = numbers.reduce { sum, element -> sum + element }
-    println(sum)
+    val simpleSum = numbers.reduce { sum, element -> sum + element }
```

Comment:
Sorry, I don't see a point in this change. It's just a usual sum, no need to complicate.

Reply:
This is to avoid the warning when launching it:

<img width="894" alt="Screenshot 2021-09-16 at 14 42 51" src="https://user-images.githubusercontent.com/78360457/133606104-dd1a5725-07b4-4265-9416-f32f2c0271fe.png">

Reply:
Good catch, I haven't checked this.